### DISCUSSION TITLE
Teams Dashboard

### DISCUSSION THREAD

**Discussion by oyi77** [super-productivity/super-productivity#2530] [General]
Hi, since i just start to search about team things, and not find anything. 
Is this app, can be done to see our full team doing? As like this, i know we could use jira, but i want to see, what my other teams doing in a daily. is there spending time wisely, or just nagging, or playing.

Is this feature available?

**Comment by johannesjo** [comment]
It's for single users, but it can be integrated with using team oriented tools like Jira or Github projects or Open Project.

**Comment by oyi77** [comment]
But if only integrated with jira, i think we cant see, what others doing
right?
Is there any possible improvement, in the near future?

Pada tanggal Sab, 25 Mar 2023 pukul 00.22 Johannes Millan <
***@***.***> menulis:

**Reply by johannesjo** [reply]
Not likely due to the limitations brought by owning your data and being able to decide where to save it. The app is meant a s personal task manager.
